Firmware/Marlin_main.cpp

```diff
@@ -7794,8 +7796,18 @@ SERIAL_PROTOCOLPGM("\n\n");
 		iPindaC = code_value_uint8();
 	}
 	else
+	{
 		iPindaC = eeprom_read_byte(&EEPROM_Sheets_base->s[iSel].pinda_temp);
+	}
+	
+	if (code_seen('A'))
+	{
+		bHasIsActive = true;
+		bIsActive = code_value_uint8() || (eeprom_read_byte(&(EEPROM_Sheets_base->active_sheet)) == iSel);
+	}
+	else
 	{
+		bIsActive = (eeprom_read_byte(&(EEPROM_Sheets_base->active_sheet)) == iSel);
```

Comment:
I can't help feel that this code can be optimised more for code size. `(eeprom_read_byte(&(EEPROM_Sheets_base->active_sheet)) == iSel);` is duplicated in both `if` and `else`.

I am nitpicking here, I don't see any issue.